core/sr-api-macros/tests/runtime_calls.rs

```diff
@@ -191,3 +192,16 @@ fn record_proof_works() {
 		&block.encode(),
 	).expect("Executes block while using the proof backend");
 }
+
+#[test]
+fn returns_mutable_static() {
+	let client = TestClientBuilder::new().set_execution_strategy(ExecutionStrategy::AlwaysWasm).build();
+	let runtime_api = client.runtime_api();
+	let block_id = BlockId::Number(client.info().chain.best_number);
+
+	let ret = runtime_api.returns_mutable_static(&block_id).unwrap();
+	assert_eq!(ret, 33);
+
+	let ret = runtime_api.returns_mutable_static(&block_id).unwrap();
+	assert_eq!(ret, 34);
```

Comment:
But is it actually right? We expect that every invocation will need to return the initial value plus one. If the value actually increases more than that then it is the sign that the wasm runtime preserves the memory contents.

```suggestion
	assert_eq!(ret, 33);
```